rtcstats.js

```diff
@@ -1,5 +1,15 @@
 'use strict';
 (function() {
+  // transforms a maplike to an object. Mostly for getStats +
+  // JSON.parse(JSON.stringify())
+  function m2o(m) {
```

Comment:
Personally I prefer writing things out verbatime, so `mapToObject`, makes it easier to read. Same with the below, it gets smaller in size, but compilers do those things, I want to read and understand easily